go-controller/pkg/node/node.go

```diff
@@ -784,6 +796,10 @@ func (n *OvnNode) checkAndDeleteStaleConntrackEntriesForNamespace(newNs *kapi.Na
 		validNextHopMACs = append(validNextHopMACs, value.([]byte))
 		return true
 	})
+
+	// TODO comment: The lines below handle already the case in which the ARPs above failed,
```

Comment:
makes total sense... if the arp wasn't successful didn't we already have a 30 second retry goroutine? are we removing that in this PR for the retry mechanism? I think that was one of those requirements for a corner case, where we regularly purge the entries to avoid the race between a crio shutdown and pod deletion or something...

Reply:
no, should not be removing the periodic retry